integration-tests/docker/test_env/cl_node.go

```diff
@@ -107,11 +105,10 @@ func NewClNode(networks []string, nodeConfig *chainlink.Config, opts ...ClNodeOp
 	return n
 }
 
-func (n *ClNode) WithTestLogger(t *testing.T) *ClNode {
+func (n *ClNode) SetTestLogger(t *testing.T) {
```

Comment:
I like this name better! I think I used With instead of Set in all the others so I will probably go back an update those at some point to match.

Reply:
yeah 👍 , I was just looking at the code and updated it. I think `set` is a more common word for this one